docs/objects.md

```diff
@@ -328,6 +328,195 @@ The formula to derive the pool's PKIprofile is as follows:
 ```
 pkiprofilenam = poolName + "-pkiprofile"
 ```
+### Markers for Avi Objects
+
+With Avi controller version 20.1.5, Avi objects can be labelled with markers. Each marker is pair of `key` : `<value1>`, `<value2>`. Each object can have multiple markers. Markers can be used to group objects at Avi controller side and provide permission to users to access these objects. Avi controller support markers to all license types from 20.1.6.
+
+With AKO version 1.5.1, AKO add markers to Avi objects. Each objects will be marked with fields of ingress/routes such as namespace, fqdn of ingress/route, service name of ingress/route etc. This will help in identifying object associate with specific ingress/route at controller side. Markers will be added to objects in SNI and EVH deployment modes.
+
+#### L4 Virtual Service
+
+L4 VS will be marked as follows:
+
+```
+clustername : <clustername>
+Namespace   : <namespace>
+ServiceName : <servicename>
+```
+
+#### L4 Pool
+
+L4 pool will be marked as follows:
+
+```
+clustername : <clustername>
+Namespace   : <namespace>
+ServiceName : <servicename>
+Port        : <port>
+```
+
+#### Shared Virtual service
+
+Shared objects will be marked with only clustername.
+
+```
+clustername : <clustername>
+```
+
+### Shared Pool group
+
+Shared poolgroup will be marked as follows:
+
+```
+clustername : <clustername>
+```
+
+#### Shared Pool
+
+Shared pool will be marked as follows:
+
+```
+clustername : <clustername>
+```
+
+#### Passthrough Virtual Service
+
+Passthrough virtual service will be marked as follows:
+
+```
+clustername : <clustername>
+```
+
+#### Passthrough Pool Group
+
+Passthrough pool group will be marked as follows:
+
+```
+clustername : <clustername>
+Host        : <fqdn>
+```
+
+#### Passthrough Pool
+
+Passthrough pool will be marked as follows:
+
+```
+clustername : <clustername>
+Host        : <fqdn>
+ServiceName : <servicename>
+```
+
+#### Advance L4 VS
```

Comment:
The gateway VSes created as part of regular `servicesAPI: true` workflow is not covered here. We should probably replace those with `Advance L4` objects, probably to `L4 Gateway VS, L4 Gateway Pool..`

Reply:
Did not understand comment. Can you please elaborate more?